README.md

```diff
@@ -16,9 +16,40 @@ In summary, our software suite addresses the multifaceted challenges associated
 
-...
+After installation, _NWBInspector_ can be used very easily from the command line. Simply run
+```bash
+    nwbinspector path/to/my/data.nwb
+```
+to inspect a single NWB file or
+```bash
+    nwbinspector path/to/my/data/dir/
+```
+to inspect all the NWB files in a given directory.
 
-TBD
+This command will produce a report of all the issues flagged by _NWB Inspector_ directly in the terminal, ordered by importance as shown below.
+[Example output]
 
+Unlike the _PyNWB_ validator, which checks for strict schema compliance, _NWBInspector_ only serves as an automatic data review aid. This means that, in an effort to catch as many potential mistakes as possible, _NWBInspector_ may raise warnings for intentional deviations from standard best practices. Conversely, if a user makes a mistake that does not violate any established best practices and it passes the _PyNWB_ validator, _NWBInspector_ might not catch that mistake. This structure leads to a trade-off between comprehensive inspection, which catches as many potential mistakes as possible, and an unobtrusive inspection process that only highlights truly problematic choices rather than intentional but unorthodox decisions.
+
+Our solution to this trade-off involves splitting the results into three tiers of warnings/errors. The first tier is for best practice violations, which are clear violations of known best practices, such as negative spike times. The second tier is for critical issues, which are potentially critical mistakes, such as zero-rate timestamps. The third tier is for best practice suggestions, which are recommendations for improving the NWB file, such as adding an informative description. This approach allows our inspection process to be as comprehensive as possible while minimizing disruptions from ignoring specific best practice suggestions.
+
+Power users of NWBInspector often seek more direct control of the inspection process and its outputs. To enable more flexibility when using _NWBInspector_, we provide a simple Python API with all the same functionality as the CLI, along with some more advanced features. Just like the CLI, the Python API can be employed to inspect individual NWB files and entire directories of NWB files with the `nwbinspector.inspect_nwbfile()` function and `nwbinspector.inspect_all()` function respectively. Additionally, the Python API allows users to inspect NWB files directly in memory with the `nwbinspector.inspect_nwbfile_object()` function. Since both the `inspect_nwbfile` and the `inspect_all` functions return generators, Python users can employ these functions in more sophisticated ways, such as implementing early stopping if a particular message is raised. These features empower advanced users to fully leverage _NWBInspector_.
```

Comment:
```suggestion
Power users of NWB Inspector often seek more direct control of the inspection process and its outputs. To enable more flexibility when using _NWBInspector_, we provide a simple Python API with all the same functionality as the CLI, along with some more advanced features. Just like the CLI, the Python API can be employed to inspect individual NWB files and entire directories of NWB files with the `nwbinspector.inspect_nwbfile()` function and `nwbinspector.inspect_all()` function respectively. Additionally, the Python API allows users to inspect NWB files directly in memory with the `nwbinspector.inspect_nwbfile_object()` function. Since both the `inspect_nwbfile` and the `inspect_all` functions return generators, Python users can employ these functions in more sophisticated ways, such as implementing early stopping if a particular message is raised. These features empower advanced users to fully leverage _NWBInspector_.
```